google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -306,6 +306,17 @@ def max_outstanding_bytes
         # @deprecated Use {#max_outstanding_bytes}.
         alias inventory_bytesize max_outstanding_bytes
 
+        ##
+        # Whether to enforce flow control at the client side only or to enforce it at both the client and
+        # the server.
+        #
+        # @return [Boolean] `true` when only client side flow control is enforced, `false` when both client and
+        # server side flow control are enforced.
+        #
+        def use_legacy_flow_control
```

Comment:
To follow Ruby conventions, the name of a boolean query method should end in `?`:

```
def use_legacy_flow_control?
```

Reply:
Thanks. Done.